resources/views/dockerfile.blade.php

```diff
@@ -17,10 +17,20 @@
     && mkdir -p storage/logs \
     && php artisan optimize:clear \
     && chown -R www-data:www-data /var/www/html \
-    && sed -i 's/protected \$proxies/protected \$proxies = "*"/g' app/Http/Middleware/TrustProxies.php \
     && echo "MAILTO=\"\"\n* * * * * www-data /usr/bin/php /var/www/html/artisan schedule:run" > /etc/cron.d/laravel; \
     if [ -d .fly ]; then cp .fly/entrypoint.sh /entrypoint; chmod +x /entrypoint; fi;
 
+# Different syntax for Laravel 11 and previous versions 
+# For setting TrustProxies
```

Comment:
The comment above makes sense when you are reading the template, but won't make as much sense when reading the output produced.  Consider using Blade comments `{{-- ... --}}` instead of Dockerfile comments.